README.md

```diff
@@ -3,7 +3,71 @@
 [![Swift 5.2](https://img.shields.io/badge/Swift-5.2-ED523F.svg?style=flat)](https://swift.org/download/)
 [![CI](https://github.com/slashmo/gsoc-swift-baggage-context/workflows/CI/badge.svg)](https://github.com/slashmo/gsoc-swift-baggage-context/actions?query=workflow%3ACI)
 
-`BaggageContext` originated in [this repo](https://github.com/slashmo/gsoc-swift-tracing), but had to be factored out so that we can depend on it in our [NIO fork](https://github.com/slashmo/swift-nio).
+`BaggageContext` is a minimal (zero-dependency) "context" library meant to "carry" baggage (metadata) for cross-cutting tools such as tracers.
+It is purposefully not tied to any specific use-case (in the spirit of the Tracing Plane paper's BaggageContext), however does not implement its own serialization scheme (today), however should enable vast majority of use cases cross-cutting tools need to support.
```

Comment:
```suggestion
It is purposefully not tied to any specific use-case (in the spirit of the [Tracing Plane paper](https://cs.brown.edu/~jcmace/papers/mace18universal.pdf)'s BaggageContext), however does not implement its own serialization scheme (today), however should enable vast majority of use cases cross-cutting tools need to support.
```

I assume not everyone knows the paper, so I think we should link it.

Reply:
Good point 👍